cordova-lib/src/cordova/plugin.js

```diff
@@ -360,6 +360,12 @@ function getPluginVariables(variables){
 
 function getVersionFromConfigFile(plugin, cfg){
     var pluginEntry = cfg.getPlugin(plugin);
+    if (!pluginEntry) {
+        var oldStylePluginId = pluginMapper[plugin];
+        if (oldStylePluginId) {
```

Comment:
Nope, it really is oldStylePluginId... Have a look at the declaration of pluginMapper.